octoprint_mrbeam/static/js/app/view-models/user-notification.js

```diff
@@ -1,44 +1,152 @@
-                },
             },
-            err_hardware_malfunction: {
+            err_hardware_malfunction_i2c: {
                 title: gettext("Hardware malfunction"),
-                text: gettext(
-                    "A possible hardware malfunction has been detected on this device. You can not start a laser job until the problem is resolved."
+                text: _.sprintf(
+                    gettext(
+                        "The device is currently not able to start any laser job.\n" +
+                            "\n" +
+                            "Please shutdown the device and check the ribbon cable following the %(opening_tag)sKnowledgebase article%(closing_tag)s.\n" +
+                            "\n" +
+                            "If this doesn’t solve the problem, please %(opening_tag_support)sopen a support ticket%(closing_tag_support)s."
+                    ),
+                    self._getKowledgebaseArticleLinkSprintf(
+                        gettext("https://support.mr-beam.org/en") +
+                            "/support/solutions/articles/43000703001-hardware-malfunction-how-to-check-the-ribbon-cable"
+                    )
+                ),
+                type: "error",
+                hide: false,
+            },
+            err_compressor_malfunction: {
+                title: gettext("Compressor malfunction"),
+                text: _.sprintf(
+                    gettext(
+                        "The device is currently not able to start any laser job.\n" +
+                            "\n" +
+                            "Please check the piping in the working area by following the %(opening_tag)sKnowledgebase article%(closing_tag)s. And restart the device.\n" +
+                            "\n" +
+                            "If this doesn’t solve the problem, please %(opening_tag_support)sopen a support ticket%(closing_tag_support)s."
+                    ),
+                    self._getKowledgebaseArticleLinkSprintf(
+                        gettext("https://support.mr-beam.org/en") +
+                            "/support/solutions/articles/43000702996-compressor-malfunction-how-to-check-the-piping"
+                    )
+                ),
+                type: "error",
+                hide: false,
+            },
+            err_one_button_malfunction: {
+                title: gettext("One Button malfunction"),
+                text: _.sprintf(
+                    gettext(
+                        "The device is currently not able to start any laser job.\n" +
+                            "\n" +
+                            "You can still shut down the device via the Mr Beam Software.\n" +
+                            "\n" +
+                            "Please %(opening_tag_support)sopen a support ticket%(closing_tag_support)s."
+                    ),
+                    self._getKowledgebaseArticleLinkSprintf(self.err)
```

Comment:
I think we need to pass an empty url, before the error code
```suggestion
                    self._getKowledgebaseArticleLinkSprintf(None,self.err)
```

Reply:
I removed the error as this not needed anymore, I also renamed the function